MODULE.bazel

```diff
@@ -8,7 +8,7 @@ module(
 
 # Lower-bound versions of direct dependencies.
 # When bumping, add a comment explaining what's required from the newer release.
-bazel_dep(name = "aspect_bazel_lib", version = "1.40.0")
+bazel_dep(name = "aspect_bazel_lib", version = "2.9.1") # py_image_layer requires 2.x for the `tar` rule.
```

Comment:
this is an important change before 1.0 actually. Would be technically breaking to increase minimum later

Reply:
yes, we should have made this change nevertheless.